src/main/java/net/masterthought/cucumber/json/support/StatusCounter.java

```diff
@@ -66,7 +56,10 @@ public int size() {
      *
      * @return final status for this counter
      */
-    public Status getFinalStatus() {
-        return finalStatus;
+    public Status getFinalStatus(boolean strict) {
+        if (strict) {
+            return counter.get(Status.PASSED) == size() ? Status.PASSED : Status.FAILED;
```

Comment:
would you be able to see pending statuses on the overview pages if the `strict` mode is on?